checks/__init__.py

```diff
@@ -22,6 +22,7 @@
 
 # 3rd party
 import yaml
+import psutil
```

Comment:
psutil is actually not available on source install without a compiler :/ 
Can you try and catch an import error and disable the developer mode accordingly ?